jetson/ra_kinematics/mrover_arm.cpp

```diff
@@ -676,40 +673,10 @@ void MRoverArm::check_joint_limits(std::vector<double> &angles) {
     }
 }
 
-double MRoverArm::joint_b_stabilizer(double angle) {
-    // If prev_angle hasn't been set yet
-    if (std::isnan(prev_angle_b)) {
-        prev_angle_b = angle;
-        return angle;
-    }
-
-    double prev_multiplier;
-
-    // If angle seems very bad
-    if (std::isnan(angle) ||
-        angle < arm_state.get_joint_limits(1)[0] ||
-        angle > arm_state.get_joint_limits(1)[1]) {
-
-        prev_multiplier = 1.0;
-    }
-    // Else if angle seems kinda bad
-    else if (std::abs(angle - prev_angle_b) > ENCODER_ERROR_THRESHOLD) {
-        prev_multiplier = JOINT_B_STABILIZE_BAD_MULTIPLIER;
-    }
-    else {
-        prev_multiplier = JOINT_B_STABILIZE_MULTIPLIER;
-    }
-
-    // Update prev angle to current angle
-    prev_angle_b = prev_multiplier * prev_angle_b + (1 - prev_multiplier) * angle;
-    return prev_angle_b;
-}
-
-
 void MRoverArm::send_kill_cmd() {
 
```

Comment:
If we're sending the kill_cmd, we might as well kill the science arm stuff too (so maybe make this a virtual function too). Would have to publish to "sa_openloop_cmd" and "/foot_openloop_cmd".